packages/cli/lib/init-global.ts

```diff
@@ -11,14 +12,26 @@ export function initGlobal() {
 
   if (workspace) {
     // Found a workspace root - hand off to the local copy of Nx
-    require(path.join(
+    const localNx = path.join(
       workspace.dir,
       'node_modules',
       '@nrwl',
       'cli',
       'bin',
       'nx.js'
-    ));
+    );
+    if (fs.existsSync(localNx)) {
```

Comment:
Is this also going to show when user forgets to npm install? If so, the message should say that as a possible cause.

Reply:
Good point. I'll update the message.